### PR TITLE
Add a file for New Ulm, MN

### PR DESCRIPTION
**Pull request by mm-sfg**
The purpose of this PR is to add a hometown file for New Ulm, MN :partly_sunny: 

Some :question: 's that I want you all to help me answer:
1. If you've been to New Ulm, what did you do? What did you think of it?
1. What restaurants did you go to?
1. Why did you go to New Ulm? If you haven't been there, why would you go?

### CODE REVIEW

**Comment by NatashaSteblay** [comment]
Never been to New Ulm. @mm-sfg

File: mm-sfg-newulm.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# New Ulm, MN
+
+### Restaurants
+
+### Places the locals like
```
Comment (cheri-tolkinen) [inline mm-sfg-newulm.md:5]: Herman the German

**Comment by mm-sfg** [comment]
> Herman the German

testing reply

**Comment by mm-sfg** [comment]
This Pull Request resolves #8

**Review by NatashaSteblay — APPROVED** [review]

**Pull request merged**